### PR TITLE
add --verbose option

### PR DESCRIPTION
**Pull request by krichprollsch**
use `--verbose` to enable debug output.
In debug mode, verbose is forced to `true`.

### CODE REVIEW

File: src/main.zig [hunk 1]
```diff
@@ -334,3 +347,18 @@ pub fn main() !void {
         },
     }
 }
+
+var verbose: bool = builtin.mode == .Debug; // In debug mode, force verbose.
```
Comment (francisbouvier) [inline src/main.zig:351]: Not a big fan of this global variable but I guess it's not possible to do without, at least if we want to keep using the std lib log function.

File: src/main.zig [hunk 2]
```diff
@@ -334,3 +347,18 @@ pub fn main() !void {
...
+fn logFn(
+    comptime level: std.log.Level,
+    comptime scope: @Type(.EnumLiteral),
+    comptime format: []const u8,
+    args: anytype,
+) void {
+    if (!verbose) {
+        // hide all messages with level greater of equal to debug level.
+        if (@intFromEnum(level) >= @intFromEnum(std.log.Level.debug)) return;
```
Comment (francisbouvier) [inline src/main.zig:360]: As `debug` is the last level I think checking an equality should be more clear. But of course like that you can't handle addition lowers levels. So it's also good to me like.

**Review by francisbouvier — APPROVED** [review]

File: src/main.zig (hunk 1 above)
Comment (krichprollsch) [reply src/main.zig:351]: Well, I think have a global var in the `main.zig` is not perfect, but ok.

File: src/main.zig (hunk 2 above)
Comment (krichprollsch) [reply src/main.zig:360]: Good point

Comment (krichprollsch) [reply src/main.zig:360]: But Since we filter here, I do prefer the `>=` to avoid more verbose messages if a lower level is added by Zig later.

**Pull request merged**